Libraries/Components/DrawerAndroid/DrawerLayoutAndroid.android.js

```diff
@@ -56,7 +56,7 @@ type Props = $ReadOnly<{|
    * );
    * ```
    */
-  drawerBackgroundColor: ColorValue,
```

Comment:
actually we need this in Props i have added it back

Reply:
We shouldn't remove this prop as users should still be able to specify a backgroundColor other than the default

Reply:
when we keep this as a part of "Props", it throws an error which says -- """element because property `drawerBackgroundColor` is missing in
props [1] but exists in `Props` [2]. [prop-missing]""". Any work around for this? @lunaleaps ?